app/models/EmailTemplate.scala

```diff
@@ -33,18 +33,18 @@ sealed trait EmailTemplate {
 }
 
 object EmailTemplate {
-  def fromNotification(emailAddress: EmailAddress, notification: Notification): Option[EmailTemplate] = {
+  def fromNotification(emailAddress: EmailAddress, notification: Notification, companyName: String): Option[EmailTemplate] = {
     val isHistoricStatement = notification.metadata.contains("statementRequestID")
     notification.fileRole.value match {
-      case "DutyDefermentStatement" if isHistoricStatement => Some(HistoricDutyDefermentStatementEmail(emailAddress, notification.eori.value))
-      case "DutyDefermentStatement" => createDutyDefermentEmailRequestParams(notification.metadata).map { params => DutyDefermentStatementEmail(emailAddress, notification.eori.value, params) }
-      case "C79Certificate" if isHistoricStatement => Some(HistoricC79CertificateEmail(emailAddress, notification.eori.value))
-      case "C79Certificate" => Some(C79CertificateEmail(emailAddress, notification.eori.value))
-      case "SecurityStatement" if isHistoricStatement => Some(HistoricSecurityStatementEmail(emailAddress, notification.eori.value))
-      case "SecurityStatement" => Some(SecurityStatementEmail(emailAddress, notification.eori.value))
-      case "PostponedVATStatement" if isHistoricStatement => Some(HistoricPostponedVATStatementEmail(emailAddress, notification.eori.value))
-      case "PostponedVATStatement" => Some(PostponedVatEmail(emailAddress, notification.eori.value))
-      case "StandingAuthority" => Some(AuthoritiesStatementEmail(emailAddress, notification.eori.value))
+      case "DutyDefermentStatement" if isHistoricStatement => Some(HistoricDutyDefermentStatementEmail(emailAddress, notification.eori.value, Map("recipientName_line1"-> companyName)))
+      case "DutyDefermentStatement" => createDutyDefermentEmailRequestParams(notification.metadata).map { params => DutyDefermentStatementEmail(emailAddress, notification.eori.value, params ++ Map("recipientName_line1"-> companyName)) }
```

Comment:
Can we add some breaks into these long lines for readability purposes?

Reply:
Done the required changes